schema.graphql

```diff
@@ -0,0 +1,291 @@
+
+  """List all the Tickets held by this address."""
+  tickets(
+    """Returns the elements in the list that come after the specified cursor"""
+    after: String
+
+    """Returns the elements in the list that come before the specified cursor"""
+    before: String
+
+    """Returns the first n elements from the list."""
+    first: Int
+
+    """Returns the last n elements from the list."""
+    last: Int
+  ): TicketConnection
+}
+
+type CENNZNode {
+  name: String
+}
+
+type CENNZNodeConnection {
+  """
+  https://facebook.github.io/relay/graphql/connections.htm#sec-Edge-Types
+  """
+  edges: [CENNZNodeEdge]
+
+  """
+  https://facebook.github.io/relay/graphql/connections.htm#sec-undefined.PageInfo
+  """
+  pageInfo: PageInfo!
+  totalCount: Int
+}
+
+type CENNZNodeEdge {
+  """https://facebook.github.io/relay/graphql/connections.htm#sec-Cursor"""
+  cursor: String!
+
+  """https://facebook.github.io/relay/graphql/connections.htm#sec-Node"""
+  node: CENNZNode
+}
+
+"""
+A date string, such as 2007-12-03, compliant with the `full-date` format outlined in section 5.6 of the RFC 3339 profile of the ISO 8601 standard for representation of dates and times using the Gregorian calendar.
+"""
+scalar Date
+
+type Event {
+  createAdditionalTickets(quantity: Int, ticketTypeId: String): Transcation
+  createNewTicketType(quantity: Int, ticketType: TicketTypeInput): Transcation
+  dateTime: Date
+  description: String
+  name: String
+  ticketStubs(
+    """Returns the elements in the list that come after the specified cursor"""
+    after: String
+
+    """Returns the elements in the list that come before the specified cursor"""
+    before: String
+
+    """Returns the first n elements from the list."""
+    first: Int
+
+    """Returns the last n elements from the list."""
+    last: Int
+  ): TicketStubConnection
+  ticketTypes: [TicketType]
+  tickets(
+    """Returns the elements in the list that come after the specified cursor"""
+    after: String
+
+    """Returns the elements in the list that come before the specified cursor"""
+    before: String
+
+    """Returns the first n elements from the list."""
+    first: Int
+
+    """Returns the last n elements from the list."""
+    last: Int
+  ): TicketConnection
+  venue: String
+}
+
+type Mutation {
+  """Used to submit a transcation to the blockchain"""
+  submitTranscation(signature: String, transcationData: String): TranscationResult!
+}
+
+type Network {
+  address(address: String!): Address
+  event(id: String!): Event
+  name: String
+  nodes(
+    """Returns the elements in the list that come after the specified cursor"""
+    after: String
+
+    """Returns the elements in the list that come before the specified cursor"""
+    before: String
+
+    """Returns the first n elements from the list."""
+    first: Int
+
+    """Returns the last n elements from the list."""
+    last: Int
+  ): CENNZNodeConnection
+}
+
+enum NetworkEnum {
+  Nikau
+}
+
+interface Node {
```

Comment:
is there a better name we can use? or is this more a blockchain thing?

Reply:
https://graphql.org/learn/global-object-identification/#node-interface